models/user.go

```diff
@@ -877,6 +879,28 @@ func UpdateUser(u *User) error {
 	return updateUser(x, u)
 }
 
+// UpdateUserCols update user according special columns
+func UpdateUserCols(u *User, cols ...string) error {
+	// Organization does not need email
+	u.Email = strings.ToLower(u.Email)
```

Comment:
Can't we iterate over cols array and with a switch only do needed operation want matching col ?

Example : https://play.golang.org/p/PFVkvT3JYg

u.Email = strings.ToLower(u.Email)